src/main/java/robot/subsystems/drivetrain/SwerveDrive.java

```diff
@@ -35,7 +35,7 @@ public SwerveDrive(boolean isFieldOriented) {
      */
     public void holonomicDrive(double forward, double strafe, double rotation) {
 
-        double[] robotHeading = getRobotHeading(forward, strafe, rotation);
+        double[] robotHeading = getRobotHeading(forward, strafe, rotation, gyro.getAngle());

```

Comment:
The `getAngle` method of the navx return the angle in degrees and not in radians